pubsub/driver/driver.go

```diff
@@ -86,14 +86,11 @@ type Topic interface {
 	IsRetryable(err error) bool
 
 	// As allows providers to expose provider-specific types.
-	//
-	// See
-	// https://github.com/google/go-cloud/blob/master/internal/docs/design.md#as
-	// for more background.
+	// See https://godoc.org/gocloud.dev#As for background information.
 	As(i interface{}) bool
 
 	// ErrorAs allows providers to expose provider-specific types for errors.
-	// See As.
+	// See https://godoc.org/gocloud.dev#As for background information.
```

Comment:
Again, I think it might be shorter and more consistent to reference the primary `As` function in this package, or duplicate the entire comment there.

Reply:
Addressed as elsewhere